CONTRIBUTING.md

```diff
@@ -432,6 +432,15 @@ advised nor expected. Valid reasons to bump the version are for example:
 - An important feature has been implemented,
 - An external project depends on one of the recent changes.
 
+## OpenAPI specification update workflow
+
+* Plan a change to the Conjur REST API
```

Comment:
```suggestion
Are you planning a change to the Conjur API? This could involve adding a new endpoint, extending an
existing endpoint, or changing the response of an existing endpoint. **When you make changes to the
Conjur API, you must also update the [Conjur OpenAPI Spec](https://github.com/cyberark/conjur-openapi-spec).**

To prepare to make a change to the Conjur API, follow the process below:
```

This provides more context for what this section is, and why someone should care about it. I think the steps below should also be more clear about the purpose of each and how to do them. For example:
1. Clone the [OpenAPI spec project](https://github.com/cyberark/conjur-openapi-spec) and create a branch.
2. Update the spec with your planned API changes and create a draft pull request; make sure it references the Conjur issue you are working on. **Note:** it is expected that the automated tests in your spec branch will fail, because they are running against the `conjur:edge` image which hasn't been updated with your API changes yet.
3. Return to your clone of the Conjur project, and make your planned changes to the Conjur API following the standard branch / review / merge workflow.
4. Once your Conjur changes have been merged and the new `conjur:edge` image has been published, rerun the automation in your OpenAPI pull request to ensure that the spec is consistent with your API changes. Have your spec PR reviewed and merged as usual.